src/System.CodeDom/src/Microsoft/CSharp/CSharpCodeGenerator.pns.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please name this file something more like CSharpCodeGenerator.NotSupported.cs; acronyms like "pns" are not easily understandable when looking through a directory.  Same goes for the VB one.